src/mono/wasm/wasm.proj

```diff
@@ -160,7 +160,7 @@
     </ItemGroup>
     <PropertyGroup>
       <PInvokeTableFile>$(ArtifactsObjDir)wasm/pinvoke-table.h</PInvokeTableFile>
-      <CMakeConfigurationEmccFlags Condition="'$(Configuration)' == 'Debug'">-g -Os -s -DDEBUG=1 -DENABLE_AOT_PROFILER=1</CMakeConfigurationEmccFlags>
+      <CMakeConfigurationEmccFlags Condition="'$(Configuration)' == 'Debug'">-g4 -Os -s -DDEBUG=1 -DENABLE_AOT_PROFILER=1</CMakeConfigurationEmccFlags>
```

Comment:
https://emscripten.org/docs/tools_reference/emcc.html#emcc-gn only describes up through -g3, and says -g is equivalent  to -g3.  Why do we need a 4?

Reply:
I think `-g3` and `-gsource-map` are likely sufficient now.

Reply:
I would update this too.